### PR TITLE
Excluded SV-230368 from the password reuse requirement for HML per ARS 5.0

### PR DESCRIPTION
**Pull request by damif5**
…RS 5.0

Signed-off-by: Dami Afolabi <dami.afolabi@icloud.com>

### CODE REVIEW

File: inspec.yml [hunk 1]
```diff
@@ -40,6 +40,7 @@ inputs:
       - "SV-230365" # IA-5 (1) d
       - "SV-230366" # IA-5 (1) d
       - "SV-230367" # IA-5 (1) d
+      - "SV-230368" # IA-5 (1) d
```
Comment (karikarshivani) [inline inspec.yml:43]: I'm not sure if it's `d`. In ARS, it seems to be `h`. @ejaronne?

**Review by karikarshivani — CHANGES_REQUESTED** [review]

File: inspec.yml (hunk 1 above)
Comment (damif5) [reply inspec.yml]: done

File: inspec.yml [hunk 2]
```diff
@@ -31,27 +31,28 @@ inputs:
...
       - "SV-230483" # AU-5 (1)
       - "SV-244543" # AU-5 (1)
       - "SV-230296" # IA-2 (5)
       - "SV-230282" # SI-6 a
-      - "SV-230359" # IA-5 (1) a
-      - "SV-230362" # IA-5 (1) b
-      - "SV-230375" # IA-5 (1) a
+      - "SV-230359" # IA-5 (1) 
+      - "SV-230362" # IA-5 (1) 
+      - "SV-230375" # IA-5 (1)
```
Comment (karikarshivani) [inline inspec.yml:55]: I see one here `SI-6 a` but everything else looks good!

**Review by karikarshivani — CHANGES_REQUESTED** [review]

File: inspec.yml (hunk 2 above)
Comment (damif5) [reply inspec.yml]: good catch...:)

**Comment by ejaronne** [comment]
not sure why the inspec.yml version isn't 1.3.1 instead of jumping to 1.3.2?

**Comment by karikarshivani** [comment]
> not sure why the inspec.yml version isn't 1.3.1 instead of jumping to 1.3.2?

Changed back to `1.3.1`

**Review by karikarshivani — APPROVED** [review]
Looks great!

**Pull request merged**